container-images/mds-agency/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
We can probably delete all of the `index.ts` files in `container-images` ... they are the AWS lambda handlers that won't be used.